generate_embeddings.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5ea23f97-50ba-4ddf-b856-a087e3ba3d55",
+   "metadata": {},
+   "source": [
+    "# Embeddings factory script\n",
+    "\n",
+    "Generate embeddings in bulk, on a per MGRS tile basis.\n",
+    "\n",
+    "Embeddings are stored into the `data/embeddings/` folder,\n",
+    "and then uploaded to `s3://clay-vector-embeddings/`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b60152d-d504-4d59-b6de-d3a3a4526e2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import warnings\n",
+    "\n",
+    "import duckdb\n",
```

Comment:
DuckDB is optional and can be removed, but it's nice to get a quick count of all the rows across the GeoParquet files (see last cell).